core/capabilities/ccip/oraclecreator/plugin.go

```diff
@@ -119,29 +118,10 @@ func (i *pluginOracleCreator) Create(config cctypes.OCR3ConfigWithMeta) (cctypes
 	destChainFamily := relay.NetworkEVM
 	destRelayID := types.NewRelayID(destChainFamily, fmt.Sprintf("%d", destChainID))
 
-	configTracker := ocrimpls.NewConfigTracker(config)
-	publicConfig, err := configTracker.PublicConfig()
-	if err != nil {
-		return nil, fmt.Errorf("failed to get public config from OCR config: %w", err)
-	}
```

Comment:
I feel like its cleaner to create this here and pass in publicConfig into `createReadersAndWriters`. Right now you create two config tracker objects

Reply:
I also like seeing all the top-level deps created in the same place if possible